Tensile/SolutionStructs.py

```diff
@@ -3709,15 +3737,25 @@ def assignDerivedParameters(state):
     # Note for these matrices LSC is same as MacroTile dim
     # MatrixInstruction rules:
     # DirectToLDS is supported for TLU=0  (make sure transposeLDS=1)
-    # LDS (load size coalesced) * LSPA must load some multiple of 256 bytes. each DirecToLds instruction provides 256 bytes
+    # LDS (load size coalesced) * LSPA must load some multiple of 256 bytes.
+    # added support for loadX2/loadx4 .
+    # x2/x4 (use x4 for better load efficiency)
+    # x2/x4 support for NT layout for f64/f32   TN layout (f16/bf16/f32/f64) with transposeLDS=1
+    # ignore x2/x4 precision < 4 bytes in NT layout
     if state["DirectToLds"]:
       if Solution.isDirectToLdsDoable(state, 'A'):
         state["DirectToLdsA"] = True
         state["LocalWriteUseSgprA"] = True
+        #1LDS buffer must be 0 for DirectToLdsA
+        state["1LDSBuffer"] = 0
```

Comment:
There's code dependent on this state in the previous section, will changing this state this late affect the results of the earlier sections?

Reply:
1LDSBuffer was introduced to support  LdsSize > maxLDS . I think the code seems like bit ugly to change state[1LDsBuffer] at later point of code..  for directToLDs=1 , there are no VGPR->LDS write so some of the code for 1LdsBuffer=1 is irrelevant. 

StoreRemapVector this might have some issue. but I believe ldsSize calcualted max of (ldsAB , ldsC); so i suppose it should work for 1LdsBuffer=0 too.

Reply:
The LocalWritePerMfma calc depends on 1LDSBuffer, is that okay as well?

Reply:
It should be OK to modify 1LDSBuffer.
LWPM calculation in SolutionStructs.py is used for debugging (set LWPM=-2). will be removed in the future.

Reply:
please consider that just reject the solution if 1LDSBuffer is enabled but not modify the user desired config.